src/releaseFiles/validateSchema.ts

```diff
@@ -0,0 +1,133 @@
+import Ajv, { ErrorObject } from "ajv";
+import ajvErrors from "ajv-errors";
+import manifestSchema from "../releaseFiles/manifest/manifest.schema.json";
+import composeSchema from "../releaseFiles/compose/compose.schema.json";
+import setupWizardSchema from "../releaseFiles/setupWizard.ts/setup-wizard.schema.json";
+import { CliError } from "../params";
+import { ReleaseFile, ReleaseFileType } from "./types";
+
+export function validateSchema(releaseFile: ReleaseFile): void {
+  const { valid, errors } = validateReleaseFile(releaseFile);
+  if (valid) return;
+
+  // If not valid, print errors and stop execution
+  throw new CliError(
+    `Invalid ${releaseFile.type}: \n${errors
+      .map(msg => `  - ${msg}`)
+      .join("\n")}`
+  );
+}
+
+/**
+ * Validates a manifest syncronously. Does NOT throw.
+ * @param manifest
+ * @returns = {
+ *   valid: false|true
+ *   errors: [
+ *     "manifest should have required property 'description'",
+ *     "manifest.image.path should be an non-empty string",
+ *   ]
+ * }
+ */
+function validateReleaseFile(
+  releaseFile: ReleaseFile
+): { valid: boolean; errors: string[] } {
+  const ajv = new Ajv({
```

Comment:
Ajv should only be initialized once in the global scope if options are not configurable